src/app.js

```diff
@@ -1 +1,44 @@
+/* eslint-disable no-console */
+/* eslint-disable max-len */
 'use strict';
+
+const styles = require('terminal-styles');
+const readline = require('readline');
+
+const randomNumber = require('./getRandomNumber');
+
+const terminal = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const { red, green, x } = styles;
+
+const game = () => {
+  terminal.question('Guess the number: ', (answer) => {
+    if (answer.length !== 4) {
+      console.log(styles`${red}Please enter a number with 4 digits${x}`);
+      game();
+    } else if (isNaN(+answer)) {
+      console.log(styles`${red}Please enter a NUMBER${x}`);
+      game();
+    } else if (answer !== [...new Set(answer)].join('')) {
+      console.log(styles`${red}Please enter a number with unique digits${x}`);
+      game();
```

Comment:
```suggestion
    if (!(/^(?:(\d)(?!.*\1)){4}$/.test(answer))) {
      console.log(styles`${red}Please enter a number with 4 unique digits${x}`);
      game();
```

You can replace all checks by one universal regexp test :) It's optional, but you can dive deeply in its structure (I've added diagram).
![regex](https://user-images.githubusercontent.com/54940595/196731665-8bb88145-b3d8-4d8f-8687-e23b6226e456.png)